language/vm/src/deserializer.rs

```diff
@@ -1128,6 +1132,7 @@ impl Opcodes {
             0x33 => Ok(Opcodes::GET_TXN_SEQUENCE_NUMBER),
             0x34 => Ok(Opcodes::GET_TXN_PUBLIC_KEY),
             0x35 => Ok(Opcodes::FREEZE_REF),
+            0x36 => Ok(Opcodes::IMM_BORROW_FIELD),
```

Comment:
I hate to ask, but: shouldn't we reorder to make this closer to `MUT_BORROW_FIELD`? Totally happy with a TODO for this, but seems like something we should do eventually.